northstar/src/runtime/island/mod.rs

```diff
@@ -107,10 +112,20 @@ impl Launcher for Island {
     where
         Self: Sized,
     {
-        Ok(Island { tx, config })
+        let (tripwire_read, tripwire_write) =
+            block(pipe).map_err(|e| Error::io("Open tripwire pipe", e))?;
+        block(|| tripwire_read.set_cloexec(true))
```

Comment:
This causes the fd to automatically be closed during execve, right? Why do we need this for our tripwire?